cmd/hactar/commands/commands.go

```diff
@@ -0,0 +1,81 @@
+package commands
+
+import (
+	"fmt"
+	"github.com/NodeFactoryIo/hactar-daemon/internal/hactar"
+	"github.com/NodeFactoryIo/hactar-daemon/internal/lotus/services"
+	"github.com/NodeFactoryIo/hactar-daemon/internal/stats"
+	"github.com/NodeFactoryIo/hactar-daemon/internal/token"
+	"github.com/mkideal/cli"
+	log "github.com/sirupsen/logrus"
+)
+
+// root command
+type rootT struct {
+	cli.Helper
+}
+
+var RootCommand = &cli.Command{
+	Argv: func() interface{} { return new(rootT) },
+	Desc: "Command line interface for Filecoin nodes monitoring deamon app",
+	Fn: func(ctx *cli.Context) error {
+		return nil
+	},
+}
+
+// start command
+type startT struct {
+	cli.Helper
+	Username string `cli:"u,username" usage:"lotus account" prompt:"Enter Lotus account"`
+	Password string `pw:"p,password" usage:"password of lotus account" prompt:"Enter Lotus password"`
+}
+
+var StartCommand = &cli.Command{
+	Name: "start",
+	Desc: "Start hactar daemon application",
+	Text: "",
+	Fn: func(ctx *cli.Context) error {
+		argv := ctx.Argv().(*startT)
+		// authenticate
+		if err := hactar.Auth(argv.Username, argv.Password); err != nil {
+			log.Error("Failed to authenticate to Lotus service.")
```

Comment:
and authentication is done on our backend

Reply:
This `hactar` package is eventually going to be the client toward our backend.